rules/integrations/aws/exfiltration_rds_using_snapshot.toml

```
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2022/02/03"
+maturity = "production"
+updated_date = "2022/02/03"
+integration = "aws"
+
+[rule]
+author = ["Elastic", "Stijn Holzhauer"]
+description = """
+This detections monitors for the sharing of a snapshot with another AWS account.
+This could indicate the exfiltration of data using
+a provided mechanism without having to actually access the Database.
```

```suggestion
This detections monitors for the sharing of a snapshot with another AWS account. This could indicate the exfiltration of 
data using a provided mechanism without having to actually access the Database.
```

You can get the proper spacing by running `toml-lint -f <this-file>` for this and other fields